build.sbt

```diff
@@ -1,12 +1,32 @@
 import Dependencies._
 
-ThisBuild / scalaVersion     := "2.13.4"
+ThisBuild / scalaVersion     := "2.13.5"
 ThisBuild / version          := "0.1.0-SNAPSHOT"
 ThisBuild / organization     := "com.example"
 ThisBuild / organizationName := "example"
 
 lazy val root = (project in file("."))
   .settings(
     name := "MoviesServer",
-    libraryDependencies += scalaTest % Test
+    libraryDependencies += scalaTest % Test,
+    libraryDependencies ++= Seq(
+      "org.typelevel" %% "cats-core"          % "2.6.0",
+      "org.typelevel" %% "cats-effect"        % "2.5.0",
+      "org.tpolecat" %% "doobie-core"         % "0.12.1",
+      "org.tpolecat" %% "doobie-postgres"     % "0.12.1",
+      "org.tpolecat" %% "doobie-specs2"       % "0.12.1",
+      "org.tpolecat" %% "doobie-hikari"       % "0.12.1",
```

Comment:
The latest version of doobie is `0.13.2`

Reply:
See https://github.com/gutiory/MoviesServer/pull/1/commits/b37e977312c549dbad94073f2917bc1b8f0bfe6a

Reply:
See https://github.com/gutiory/MoviesServer/pull/1/commits/b37e977312c549dbad94073f2917bc1b8f0bfe6a